skeleton/default.nix

```diff
@@ -15,23 +15,72 @@
   }
 }:
 with obelisk;
-project {} ({ pkgs, ... }: {
+let args = {
+  # You must accept the Android Software Development Kit License Agreement at
+  # https://developer.android.com/studio/terms in order to build Android apps.
+  # Uncomment this to indicate your acceptance:
+  android_sdk_accept_license = true;
+  allowUnfree = true;
+}; in
+
+project args ({ pkgs, ... }: {
   name = "skeleton";
+  android = {
+    executable = ps: exes: (exes (ps.frontend)).frontend;
+    executableName = "obelisk-skeleton";
+    applicationId = "systems.obsidian.obelisk.examples.minimal";
+    displayName = "Obelisk Minimal Example";
+  };
+  ios = {
+    executable = ps: exes: (exes (ps.frontend)).frontend;
+    executableName = "obelisk-skeleton";
+    bundleIdentifier = "systems.obsidian.obelisk.examples.minimal";
+    bundleName = "Obelisk Minimal Example";
+  };
+  web = {
+    executable = ps: exes: (exes (ps.frontend)).frontend;
+  };
   extraArgs = {
     staticFiles = ./static;
   };
   src = ./.;
   inputThunks = pkgs.obsidianCompilers.thunkSets.common ++ [
     pkgs._dep.source.aeson-1541
+    pkgs._dep.source.android-activity
   ];
   shells = ps: with ps; [
     backend
   ];
   overrides = [
+    ({ config, pkgs, lib, ... }: {
+      config.enableShared = if pkgs.stdenv.targetPlatform.isiOS then lib.mkForce false else true;
+      config.enableStatic = lib.mkForce true;
+    })
     ({pkgs, lib, config, ... }: {
-        packages.obelisk-run.components.library.build-tools = with pkgs; [
-          iproute
-        ];
+      packages.obelisk-run.components.library.build-tools = with pkgs; [
+        iproute
+      ];
+
+      packages.reflex-dom = {
+	      flags = {
+	        webkit2gtk = if (pkgs.stdenv.targetPlatform.isAndroid) then lib.mkForce false else true;
+	      };
+      };
+
+      packages.frontend.components.exes.frontend = {
+        frameworks = if (!pkgs.stdenv.targetPlatform.isiOS && pkgs.stdenv.targetPlatform.isDarwin) then [ pkgs.darwin.apple_sdk.frameworks.CoreFoundation ] else [ ];
+        postInstall = lib.optionalString (pkgs.stdenv.hostPlatform.isDarwin) ''
+          mkdir -p $out/obelisk-skeleton.app
+          cp -r obelisk-skeleton $out
+          cp $out/bin/obelisk-skeleton $out/obelisk-skeleton.app
+        '';
+      };
+      packages.jsaddle-wkwebview.src = (thunkSource ../dep/jsaddle) + "/jsaddle-wkwebview";
+      packages.jsaddle-wkwebview.components.library = {
+        frameworks =
+          if (pkgs.stdenv.targetPlatform.isiOS) then lib.mkForce [ pkgs.darwin.iosSdkPkgs.sdk pkgs.darwin.apple_sdk.frameworks.CoreFoundation ]
+          else [ pkgs.darwin.apple_sdk.frameworks.CoreFoundation ];
+      };
```

Comment:
Was getting webkit2gtk eval failures on android so cargo-culted this section from https://github.com/reflex-frp/reflex-platform/blob/4fccfb196ac724695a5006a456b2125eeff331e5/example/default.nix#L62

We definitely don't want this re-done on every project. 
Something more like, shoved into a trivially importable overlay on reflex-platform?
Could maybe have it implicitly applied by obelisk, but then what happens on module conflicts?

Reply:
I think the linked code should *always* be applied in obelisk (not so much in reflex-platform). If folks need to overwrite it they can use "lib.mkOverride" for a higher value than "lib.mkForce". I'd much rather this be a *default* instead of some documentation or comments anywhere

Reply:
Well yes but a default on obelisk or on the obelisk skeleton?